build.gradle.kts

```diff
@@ -365,7 +365,7 @@ subprojects {
     options.locale = "en_US"
 
     val projectName = project.name
-    if (projectName == "common" || projectName == "api" || projectName == "client-java") {
+    if (projectName == "common" || projectName == "api" || projectName == "client-java" || projectName == "filesystem") {
```

Comment:
The project name is `filesystem-hadoop3`?

Reply:
You may need to build the project locally. I think you may need to add more Java docs if the configuration is complete.

Reply:
Besides,  Task java doc is a not dependencis of `build` task, you may need to aad the following in the `filesystem-hadoop3`

![image](https://github.com/datastrato/gravitino/assets/15794564/dad7a042-2d30-4f50-b4c5-4491158c52aa)


```kotlin
tasks.build {
  dependsOn("javadoc")
}
```

Reply:
Have tried `./gradlew clean build -x test`, it seems that there is no requirement to add more java docs, but I decide to add some more docs by myself.
![image](https://github.com/datastrato/gravitino/assets/26177232/5cff3a16-ab06-4d24-bfde-9937e3388df9)

Reply:
You need to assign the JDK version to 17.

./gradlew clean build -x test -PjdkVersion=17

Reply:
Got it, thanks